lib/tree.rb

```diff
@@ -1,38 +1,57 @@
 class NoApplesError < StandardError; end
 
 class Tree
-  attr_#fill_in :height, :age, :apples, :alive
+  attr_accessor :height, :age, :apples, :alive
 
-  def initialize
+  def initialize(age, apples=[], alive, height)
+    @age = age
+    @apples = apples
+    @alive = alive
+    @height = height
   end
 
-  def age!
+  def age! # I'm not sure why this has a bang
+    if @age == 50
+      @alive = false
+    elsif
+      @age = @age + 1
```

Comment:
`@age += 1` is the shorthand equivalent.

BTW, I appreciate that it doesn't continue to age after it has died. 👍